br-ext/configs/toolchain-aarch32

```diff
@@ -2,8 +2,8 @@ BR2_TOOLCHAIN_EXTERNAL=y
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
 BR2_TOOLCHAIN_EXTERNAL_PATH="%TOP_DIR%/toolchains/aarch32"
 BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="arm-linux-gnueabihf"
-BR2_TOOLCHAIN_EXTERNAL_GCC_8=y
-BR2_TOOLCHAIN_EXTERNAL_HEADERS_4_19=y
+BR2_TOOLCHAIN_EXTERNAL_GCC_9=y
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_4_20=y
```

Comment:
Better move to BR2_TOOLCHAIN_EXTERNAL_HEADERS_5_5=y since supported in Buildroot 2020.08.

Reply:
```
>>> toolchain-external-custom  Configuring
Incorrect selection of kernel headers: expected 5.5.x, got 4.20.x
make[2]: *** [package/pkg-generic.mk:240: /home/jerome/work/optee_repo_qemu/out-br/build/toolchain-external-custom/.stamp_configured] Error 1
```

IIRC, `BR2_TOOLCHAIN_EXTERNAL_HEADERS_x_y` has to do with the version of the kernel headers that the Glibc was build against, I mean the Glibc inside the cross compiler folder. Here with GCC 9.2 downloaded from the Arm web site, it is 4.20 (as shown below) so I think we have no choice here.
```
$ grep LINUX_VERSION_CODE toolchains/aarch32/arm-none-linux-gnueabihf/libc/usr/include/linux/version.h
#define LINUX_VERSION_CODE 267277
```
267277 = 0x04140D in other words v4.20.13.

Reply:
Indeed, sorry, i mixed-up the config switches.